google/cloud/aiplatform/models.py

```diff
@@ -48,8 +57,13 @@
     env_var as gca_env_var_compat,
 )
 
+from google.cloud.aiplatform.constants import prediction as prediction_constants
+
 from google.protobuf import field_mask_pb2, json_format
 
+if TYPE_CHECKING:
```

Comment:
The modules imported by LocalModel import CPR extra packages such as fastapi, uvicorn, and docker which require users to install SDK with prediction extra packages.
Without this, any users who import any modules in the SDK need to install CPR extra packages since Model is in google/cloud/aiplatform/__init__.py. I want to avoid CPR extra packages becoming hard dependencies.